### ISSUE TITLE
Using 'inheritHeadingDepth: True' in MkDocs config

### ISSUE THREAD

**Issue by mollieswenson** [cmacmackin/markdown-include#17]
I have the markdown-include extension working with MkDocs, but when I try to add the `inheritHeadingDepth: True` config value, I get an error. I'm guessing I just don't have the config syntax right. Can anyone advise? Thanks :)

From my mkdocs.yml

```
markdown_extensions:
    - markdown_include.include:
        base_path: docs
        inheritHeadingDepth: True
```
Config instructions: https://github.com/cmacmackin/markdown-include#configuration
I have tried formatting the YML several different ways with no luck. 

Debug says:  
```
DEBUG   -  Config value: 'markdown_extensions' = [{}, u'extra', u'admonition', {}, u'sane_lists']
```

And the error:

```
ERROR   -  Config value: 'markdown_extensions'. Error: inheritHeadingDepth
[E 190516 17:55:31 ioloop:801] Exception in callback <bound method type.poll_tasks of <class 'livereload.handlers.LiveReloadHandler'>>
    Traceback (most recent call last):
      File "c:\python27\lib\site-packages\tornado\ioloop.py", line 1229, in _run
        return self.callback()
      File "c:\python27\lib\site-packages\livereload\handlers.py", line 67, in poll_tasks
        filepath, delay = cls.watcher.examine()
      File "c:\python27\lib\site-packages\livereload\watcher.py", line 73, in examine
        func and func()
      File "c:\python27\lib\site-packages\mkdocs\commands\serve.py", line 107, in builder
        site_dir=site_dir
      File "c:\python27\lib\site-packages\mkdocs\config\base.py", line 210, in load_config
        "Aborted with {0} Configuration Errors!".format(len(errors))
    ConfigurationError: Aborted with 1 Configuration Errors!
```

**Comment by jobbyworld** [comment]
Same problem but fixed, 
It Seems the pip version is not up to date
pip install git+https://github.com/cmacmackin/markdown-include.git

**Comment by cmacmackin** [comment]
This fix is included in v0.6.0.
